src/WalkingModule/app/robots/iCubGazeboV3/dcm_walking/joypad_control/tasks/regularization.ini

```diff
@@ -0,0 +1,26 @@
+robot_velocity_variable_name    "robot_velocity"
+kp                              (5.0, 5.0, 5.0,
+                                 5.0, 5.0, 5.0, 5.0,
+                                 5.0, 5.0, 5.0, 5.0,
+                                 5.0, 5.0, 5.0, 5.0, 5.0, 5.0,
+                                 5.0, 5.0, 5.0, 5.0, 5.0, 5.0)
+
+states          ("stance", "walking")
+sampling_time   0.01
+settling_time   0.5
+
+[stance]
+name    "stance"
```

Comment:
Actually, this is the way in which the [`MultiStateWeightProvider`](https://ami-iit.github.io/bipedal-locomotion-framework/classBipedalLocomotion_1_1ContinuousDynamicalSystem_1_1MultiStateWeightProvider.html#a16cd88a66f84360f7e54929e88e28021) works. The provider wants a vector named `stances` that contains the name of the groups associated to each state. Then for each state, we specify a name and the weight itself.

Reply:
> Can the name of the group be different from the name parameter?

Yes

> Do you envision some case where this could be useful?

For the time being, I don't see the reason why they should be different.

Reply:
https://github.com/robotology/walking-controllers/pull/118/commits/2cf39b7e1586a9fac14d12d318809e0ab382756a and https://github.com/robotology/walking-controllers/pull/118/commits/85c278fe26fe10c00924df93c405b4fd929061dc changes the names of the groups using uppercase. This should reduce the misunderstanding